10up/classifai
# Add Language support for OpenAI title and excerpt generation

### Description of the Change
This PR adds Language Settings that applies to both Title and Excerpt generation.
<img width="1479" alt="Screen Shot 2023-07-14 at 11 40 23 PM" src="https://github.com/10up/classifai/assets/3365507/5c50c85f-4962-4a59-bf35-358e7d4f9480">



Relates to [#150](https://github.com/10up/classifai/issues/150)

### How to test the Change

Admin should get results in the preferred language.
Switch the following when testing user cases -
- The classic editor/ The Gutenberg editor
- Use different language

#### Test
 - [ ] Visit the edit post and click on the 'Generate Titles' button 
 - [ ] Visit the edit post and click on the 'Generate/ Re-Generate Excerpt' button 
 


### Changelog Entry

> Added - Add Language support for OpenAI title and excerpt generation

### Credits

Props @phpbits @ravinderk


### Checklist:
